opencti-platform/opencti-front/src/utils/tests/test-render.tsx

```diff
@@ -93,13 +94,16 @@ interface TestRenderOptions {
  */
 const testRender = (ui: ReactNode, options?: TestRenderOptions) => {
   const { relayConfig, userContext } = options ?? {};
-  return render(ui, {
-    wrapper: ({ children }) => (
-      <ProvidersWrapper relayConfig={relayConfig} userContext={userContext}>
-        {children}
-      </ProvidersWrapper>
-    ),
-  });
+  return {
+    user: userEvent.setup(),
```

Comment:
could you make a test example where you use it please ?

Reply:
I can have a look to find a place where it could be used. In any case I have written an example using it in the documentation I am writing

Reply:
yes, don't worry. it was more a reaction than an actionnable comment 😅